portfolio/src/test/java/com/google/sps/CommentStorageTest.java

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class CommentStorageTest {
+
+  @Test
+  public void testAddComment() {
```

Comment:
Note that testing is not required for the week 3 project. It will be covered in week 5.
I would approve a potential version of this PR without it having any tests.
But it's very cool that you are adding tests, and I'd be happy to also give feedback there if you decide to go for it.